Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -274,8 +274,9 @@ private class SyncNowSetting: WithAccountSetting {
             return nil
         }
 
-        let label = NSLocalizedString("Last synced: %@", comment: "Last synced time label beside Sync Now setting option. Argument is the relative date string.")
-        let formattedLabel = String(format: label, NSDate.fromTimestamp(timestamp).toRelativeTimeString())
+        let formatter = NSDateFormatter();
```

Comment:
Nit: Creating NSDateFormatter instances is considered 'expensive' and should be done as infrequently as possible. Since this is a computer property we will be creating a new instance each time we reference status. I'd suggest moving this into a lazy var like so:

```swift
private lazy var timestampFormatter: NSDateFormatter = {
    let formatter = NSDateFormatter()
    formatter.dateFormat = "yyyy-MM-dd HH:mm:ss"
    return formatter
}()
```